dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -3,18 +3,31 @@
 mkdir -p /etc/frr
 mkdir -p /etc/supervisor/conf.d
 
+MGMT_FRAMEWORK_CONFIG=$(sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["frr_mgmt_framework_config"]')
 CFGGEN_PARAMS=" \
     -d \
     -y /etc/sonic/constants.yml \
-    -t /usr/share/sonic/templates/supervisord/frr_vars.j2 \
+    -t /usr/share/sonic/templates/frr_vars.j2 \
     -t /usr/share/sonic/templates/supervisord/supervisord.conf.j2,/etc/supervisor/conf.d/supervisord.conf \
-    -t /usr/share/sonic/templates/bgpd/bgpd.conf.j2,/etc/frr/bgpd.conf \
+    -t /usr/share/sonic/templates/supervisord/critical_processes.j2,/etc/supervisor/critical_processes \
     -t /usr/share/sonic/templates/zebra/zebra.conf.j2,/etc/frr/zebra.conf \
-    -t /usr/share/sonic/templates/staticd/staticd.conf.j2,/etc/frr/staticd.conf \
     -t /usr/share/sonic/templates/frr.conf.j2,/etc/frr/frr.conf \
     -t /usr/share/sonic/templates/isolate.j2,/usr/sbin/bgp-isolate \
     -t /usr/share/sonic/templates/unisolate.j2,/usr/sbin/bgp-unisolate \
 "
+if [ -n "$MGMT_FRAMEWORK_CONFIG" ] && [ "$MGMT_FRAMEWORK_CONFIG" == "true" ]; then
+    CFGGEN_PARAMS+=" \
+        -t /usr/local/frr/staticd/staticd.conf.j2,/etc/frr/staticd.conf \
+        -t /usr/local/frr/bgpd/bgpd.conf.j2,/etc/frr/bgpd.conf \
+        -t /usr/local/frr/bfdd/bfdd.conf.j2,/etc/frr/bfdd.conf \
+        -t /usr/local/frr/ospfd/ospfd.conf.j2,/etc/frr/ospfd.conf \
+    "
+else
+    CFGGEN_PARAMS+=" \
+        -t /usr/share/sonic/templates/bgpd/bgpd.conf.j2,/etc/frr/bgpd.conf \
+        -t /usr/share/sonic/templates/staticd/staticd.conf.j2,/etc/frr/staticd.conf \
```

Comment:
can you move this in line 6, then only generate bfdd, osfpd for MGMT_FRAMEWORK_CONFIG

Reply:
but for MGMT_FRAMEWORK_CONFIG, staticd.conf and bgpd.conf are different from existing one. if using as one file, a flag need to be added in those files to differentiate the cases

Reply:
@lguohan the template files for community bgpcfgd are present in the folder "/usr/share/sonic/templates" but the template files for new frrcfgd is copied to "/usr/local/frr/", so, there is a path difference here, so, we cant move to line 6, makes sense?